app/stats/views.py

```diff
@@ -100,35 +110,49 @@ def retrieve_annual_chart(from_year = 2009, to_year = date.today().year):
         fontSize = 18,
     )
 
-def retrieve_monthly_chart(from_year = 2017, to_year = date.today().year):
```

Comment:
As described before, I think these would be better off as objects in their own right. Some kind 'chart' object, with a defined API, like 

```
chart = Chart(db_connection)

# run a query for this kind of chart, to get back a dataframe-like result
dataframe = chart.query(params)

# build a chart object, returning an altair chart that can be updated to change properties
altair_chart = chart.build_chart()

# render chart return a render PNG file when a device does not have javascript. Or vegaspec JSON to be rendered on the front in browser for when it does
chart.render(altair_chart)

chart.render
```

Because you only have two chart, you might be able to get away without needing a class hierarchy with abstract charts.


```